File: New folder/backend/bots/template.py
def choose_template(sections, template_name):
    # Simple example: injects section ordering based on template name
    if template_name == "modern":
        ordered_sections = ["summary", "skills", "experience", "education", "certifications"]
    else:
        ordered_sections = ["summary", "experience", "education", "skills", "certifications"]
    return {section: sections.get(section, "") for section in ordered_sections}

File: New folder/backend/bots/template.js
module.exports = {
    applyTemplate: (sections, templateName) => {
        // Reorder sections based on template
        let orderedSections = templateName === "modern" ?
            ["summary", "skills", "experience", "education", "certifications"]
            : ["summary", "experience", "education", "skills", "certifications"];
        let newSections = {};
        orderedSections.forEach(key => {
            if (sections[key]) newSections[key] = sections[key];
        });
        return newSections;
    }
};